watershed_workflow/triangulation.py

```diff
@@ -194,7 +176,7 @@ def triangulate(hucs,
     fdata = [[int(i) for i in f] for f in nodes_edges.edges]
     info.set_facets(fdata)
 
-    if river_corrs is not None:
+    if not river_corrs == None:
```

Comment:
revert this, `if river_corrs is not None` is clearer than `if not river_corrs == None`